> Could you paste your whole error message, because I think I was getting same error.
seasonmac@MBP servo % ./mach build --dev
Traceback (most recent call last):
  File "/workspace/github/servo/./mach", line 46, in <module>
    main(sys.argv)
  File "/workspace/github/servo/./mach", line 27, in main
    mach = mach_bootstrap.bootstrap(topdir)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/github/servo/python/mach_bootstrap.py", line 269, in bootstrap
    mach.load_commands_from_file(os.path.join(topdir, path), module_name)
  File "/workspace/github/servo/python/mach/mach/main.py", line 262, in load_commands_from_file
    spec = importlib.util.spec_from_file_location(module_name, path)
           ^^^^^^^^^^^^^^
AttributeError: module 'importlib' has no attribute 'util'

--- a/python/mach/mach/main.py
+++ b/python/mach/mach/main.py
@@ -10,7 +10,7 @@ from __future__ import absolute_import, print_function, unicode_literals
 import argparse
 import codecs
 import errno
-import importlib
+import importlib.util
 import logging
 import os
 import sys

